lib/standard/string_nit.c

```diff
@@ -20,3 +20,57 @@ long native_int_length_str(long recv){
 void native_int_to_s(long recv, char* str, long buflen){
 	snprintf(str, buflen, "%ld", recv);
 }
+
+// Length of a UnicodeChar
+long u8char_length(uint32_t recv){
+	if(recv <= 127)
+		return 1;
+	if(recv <= 2047)
+		return 2;
+	if(recv <= 65535)
+		return 3;
+	if(recv <= 2097151)
+		return 4;
+	// Bad UTF-8 code point
+	return 1;
+}
```

Comment:
How does a pure nit implementation behave here? It should not be that slower with non-polymorphic integers. It may be even faster since there is no FFI indirection